manageiq-operator/pkg/helpers/miq-components/orchestrator.go

```diff
@@ -192,7 +190,7 @@ func OrchestratorDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*
 			},
 			corev1.EnvVar{
 				Name:  "GUID",
-				Value: uuid.New().String(),
+				Value: string(cr.GetUID()),
```

Comment:
What is this value? Is this something openshift assigns?

Reply:
All objects in kubernetes get a UID assigned by kube (in the metadata), which is a UUID, and is not modifyable by end-users.